calico/felix/test/test_fetcd_api.py

```diff
@@ -0,0 +1,163 @@
+# -*- coding: utf-8 -*-
+# Copyright 2014, 2015 Metaswitch Networks
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+felix.test.test_etcd_api
+~~~~~~~~~~~
+
+Top level tests for EtcdAPI.
+"""
+import logging
+import socket
+import gevent
+from etcd import EtcdException
+from mock import Mock, call, patch
+from calico.datamodel_v1 import key_for_status, key_for_uptime
+from calico.felix.fetcd import EtcdAPI
+from calico.felix.test.base import BaseTestCase
+
+
+_log = logging.getLogger(__name__)
+
+# To make testing continuous status reporting more convenient,
+# we speed up sleeping.

```

Comment:
This code is not inside a function/class, so importing this module will cause sleeping to speed up, perhaps unexpectedly - this could really catch someone out. 

Would be better to move this into a function, and then the single test that needs it can call that.

Also, do we need to set it back again afterwards?